### PR TITLE
Toolbar and headline

### PR DESCRIPTION
**Pull request by NduatiKagiri**
In this branch I:

- Setup all the project files
- Designed the toolbar
- Designed the headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nduati Kagiri</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <nav>
```
Comment (danielufeli) [inline index.html:10]: - [ ] Nice job with the header, kindly add the logo and nav in a header tag for better use of semantics.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+              <h5>I’am a Software Developer</h5>
+            </div>
+            <div class="me-desc">
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</p>
+            </div>
+          </div>
+          <div class="headline-social">
+            <ul>
+              <li><img src="assets/img/github.png" alt=""></li>
```
Comment (danielufeli) [inline index.html:46]: - [ ] Kindly add links to the social icons.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi John,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️
- Professional Readme ✔️
- Nice use of media queries ✔️

## Required Changes ♻️
_Check the comments under the review._

 Alt text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @NduatiKagiri 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ 🎯 Implemented previous code reviewer requested changes perfectly
- ✔️ 🎯 Perfect design implementation (pretty similar to the Figma design)
- The code is top-notch! 💯 and has a huge potential.

> **Reviewer Note**: The Student has implemented the design perfectly. He has the potential to be a great Software Engineer 💼.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**